README.md

```diff
@@ -26,30 +26,34 @@ The pipeline exposes the following standard settings:
 * `pipe_events.pipeline_bucket`: GCS bucket where all final files generated by this pipeline may be stored to.
 * `pipe_events.pipeline_dataset`: BigQuery dataset containing various tables used in this pipeline.
 
-In addition to this, the following custom settings are required for this
-pipeline, and come with default values:
+In addition to this, the following custom settings are required for this pipeline, and come with default values:
 
 * `INFERENCE_BUFFER_DAYS`: Global airflow variable which determines the amount of days to reprocess because nnet scores might be avaialble. Defaults to `7`.
 * `pipe_events.events_dataset`: BigQuery dataset which will contain the published events. Defaults to `EVENTS_DATASET`.
 * `pipe_events.source_dataset`: BigQuery dataset which contains the different tables that are read to produce the summarized events. Defaults to `PIPELINE_DATASET`.
 * `pipe_events.gaps.source_table`: BigQuery table to read gaps from. Defaults to `position_messages_`.
 * `pipe_events.gaps.events_table`: BigQuery table to publish gap events to. Defaults to `published_events_gaps`.
 * `pipe_events.gaps.segment_vessel`: BigQuery table containing segment information, used to calculate gaps. Defaults to `segment_vessel`.
+* `pipe_events.gaps.vessel_info`: BigQuery table to read vessel information from. Defaults to `vessel_info`.
```

Comment:
@andres-arana why are you defining a `vessel_info` variable for each pipe instead of having a global `pipe_events.vessel_info`, the events should be for a given dataset, so the vessel_info should be the same for all. Also in the future, when we change to the vessel database all will read from the same place.

Reply:
@enriquetuya the pipeline already supports abstracting away variables to an upper level if you want to set it up that way, so basically if you add a setting to the global `pipe_events` variable it cascades to the `pipe_events.*` subpipelines. I should add some docs pointing that up, though.